kmr_mg-ud-test.conllu

```diff
@@ -9442,8 +9442,7 @@
 6	gelek	gelek	DET	det	Number=Plur|PronType=Ind	7	det	_	_
 7	Spor	spor	NOUN	n	Case=Nom|Definite=Def|Gender=Fem|Number=Plur	10	nsubj	_	_
 8	hatin	hatin	AUX	vblex	Mood=Ind|Number=Plur|Person=3|Tense=Past|VerbForm=Fin	10	aux	_	_
-9	pêk	pêk	ADV	adv	_	10	compound:lvc	_	_
-10	anîn	anîn	VERB	vblex	Mood=Ind|Number=Plur|Person=3|Tense=Past|VerbForm=Fin	0	root	_	SpaceAfter=No
+10	pêk anîn	pêk anîn	VERB	vblex	Typo=Yes|Mood=Ind|Number=Plur|Person=3|Tense=Past|VerbForm=Fin	0	root	_	SpaceAfter=No|CorrectForm=pêkanîn
```

Comment:
This is okay now. The word infinitive form is "pêk anîn". And there is a typo in the sentence as it is passive voice.

Reply:
If the correct spelling is "pêkanîn" and the author split it to "pêk anîn", then the two "words" should keep their lines 9 and 10, but 10 should be attached to 9 via the [goeswith](https://universaldependencies.org/u/dep/goeswith.html) relation.

Reply:
So which part should the typo and correct form be specfied? "pêk" or "anîn"?

Reply:
In here it is written as single part https://github.com/UniversalDependencies/UD_Kurmanji-MG/blob/master/kmr_mg-ud-test.conllu#L6669

Reply:
Ah, I see, so Kurdish has [registered exceptions to allow words with spaces](https://github.com/UniversalDependencies/tools/blob/master/data/tokens_w_space.kmr). In that case, [goeswith](https://universaldependencies.org/u/dep/goeswith.html) would not apply and your line 9445 would be correct.

On the other hand, if _pêk anîn_ is correctly spelled _pêkanîn_ (without the space), it is questionable why we would want the exception and why we would want to allow this as a "word with space". Also, shouldn't the lemma then be spelled without the space, too?

Reply:
> Also, shouldn't the lemma then be spelled without the space, too?

No. Because [pêk anîn](https://ku.wiktionary.org/wiki/p%C3%AAk_an%C3%AEn) is written seperately but the verbs that has multi chunk are written together in passive voice and causitive verbs sentence.

Reply:
So should i replace with following?

```
10	pêk anîn	pêk anîn	VERB	vblex	Typo=Yes|Mood=Ind|Number=Plur|Person=3|Tense=Past|VerbForm=Fin	0	root	_	SpaceAfter=No|CorrectForm=pêkanîn
```

Reply:
Here is examples

Min mala xwe paqij kir. Active voice it is written seperatly.
Mala min hat paqijkirin. Passive voice it is written together.
Min mala xwe da paqijkirin. Causitive verbs it is written together.

Reply:
I specified as single word